functional_testing/zrobot_templates/framework/ovc_utils/utils.py

```diff
@@ -52,6 +52,9 @@ def create_account(self, **kwargs):
     def create_cs(self, **kwargs):
         return self.handle_blueprint('vdc.yaml', **kwargs)
```

Comment:
Please pass these params here so u don't have to pass it in the cloudspace tests
key=self.key, openvcloud=self.openvcloud, vdcusers=self.vdcusers